cmd/uplink/cmd/cp.go

```diff
@@ -84,20 +82,30 @@ func upload(ctx context.Context, src fpath.FPath, dst fpath.FPath, showProgress
 		return fmt.Errorf("source cannot be a directory: %s", src)
 	}
 
-	metainfo, streams, err := cfg.Metainfo(ctx)
+	project, err := cfg.GetProject(ctx)
 	if err != nil {
 		return err
 	}
+	defer func() {
+		err = project.Close()
```

Comment:
This must be `err := project.Close()`, i.e. with the semicolon. Otherwise, it will overwrite and lose the actual `err` returned by this method.